installer/README.md

```diff
@@ -458,6 +458,22 @@ Kubernetes objects, such as your TLS certificate or connection secrets.
 kubectl create namespace gitpod
 ```
 
+# I need to specify the load balancer - what do I do?
+
+By default, this will create a `proxy` service with `ServiceType: LoadBalancer`.
+If you need to specify your own load balancer, you can do this easily by changing
+your Installer configuration:
+
+```yaml
+meta:
+  ingress:
+    enabled: false
+```
+
+This will set the `ServiceType` to `ClusterIP` allowing you to create your own
+ingress controller - click [here](https://gist.github.com/MrSimonEmms/480e88c149f6e725c727cc1f20b21474)
```

Comment:
Do we have a better, GP-owned place to put this?

Reply:
How about somewhere in this repo under the `contrib/` folder?

Reply:
That's a good idea